CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.2.149] - 2024-01-31
```

Comment:
Please, remove this title. The release script alters the changelog automatically during the release flow. Just keep the added section under the unreleased one.